srcpkgs/optimus-manager/template

```diff
@@ -0,0 +1,58 @@
+# Template file for 'optimus-manager'
+# Originally created by Askannz, ported to Void by Hyper-KVM
+
+pkgname=optimus-manager
+version=1.2.2
+revision=1
+depends="python3 elogind mesa-demos xrandr python3-dbus"
+makedepends="git"
+hostmakedepends="python3 python3-setuptools"
+wrksrc="optimus-manager"
+archs="x86_64"
+build_style=python3-module
+short_desc="GPU switching for Optimus laptops"
+maintainer="Hyper-KVM <hyperkvmx86@gmail.com>"
+homepage="https://github.com/Askannz/optimus-manager"
+license="MIT"
+tags="python"
+
+do_fetch() {
+	git clone https://github.com/Hyper-KVM/${pkgname}.git
+}
+
+do_build() {
+	cd ${wrksrc}
+	python3 setup.py build
+}
+
+do_install() {
+	install="${wrksrc}/package/optimus-manager.install"
+	cd "${wrksrc}"
+	vinstall modules/optimus-manager.conf 644 usr/lib/modprobe.d/
+	vinstall runit/optimus-manager/run 755 etc/sv/optimus-manager
+	vinstall optimus-manager.conf 644 usr/share/
+	vinstall var/startup_mode 644 var/lib/optimus-manager/
+	vinstall var/requested_mode 644 var/lib/optimus-manager/
+	vbin scripts/prime-switch-boot
+	vbin scripts/prime-switch
+	vbin scripts/prime-offload
+	vinstall login_managers/sddm/20-optimus-manager.conf 644 etc/sddm.conf.d/
+	vinstall login_managers/lightdm/20-optimus-manager.conf 644 etc/lightdm/lightdm.conf.d/
+	vinstall config/xorg-intel.conf 644 etc/optimus-manager/
+	vinstall config/xorg-nvidia.conf 644 etc/optimus-manager/
+	vinstall config/xsetup-intel.sh 755 etc/optimus-manager/
+	vinstall config/xsetup-nvidia.sh 755 etc/optimus-manager/
+	vinstall config/xsetup-hybrid.sh 755 etc/optimus-manager/
+	vinstall config/nvidia-enable.sh 755 etc/optimus-manager/
+	vinstall config/nvidia-disable.sh 755 etc/optimus-manager/
+	vlicense LICENSE
+	python3 setup.py install --root="${wrksrc}" --optimize=1 --skip-build
```

Comment:
Point of build style is that setup.py is called with right settings, so renaming egg-info shouldn't be necessary. Is there anything wrong with using default `do_build` and `do_install` and installing extra things in `post_install`?